packages/babel-helper-builder-react-jsx-experimental/src/index.js

```diff
@@ -862,18 +863,15 @@ You can set \`throwIfNamespace: false\` to bypass this warning.`,
 
     let props = [];
     const objs = [];
-    const found = Object.create(null);
 
     for (const attr of attribs) {
       const name =
         t.isJSXAttribute(attr) &&
         t.isJSXIdentifier(attr.name) &&
         attr.name.name;
 
-      if (name === "__source" || name === "__self") {
```

Comment:
this is part of the code that will be removed in Babel 8 anyway, so I didn't have to provide any extra TODO comments or anything here

Reply:
I'm not sure I follow: why would this be removed? I think we'll keep the classic runtime, and just make the automatic one the default.

Reply:
Yeah - this is just in the compat path that handles `useSpread` and `useBuiltIns` options. It's OK to still throw this error when one uses automatic runtime.